bears/haskell/HaskellLintBear.py

```diff
@@ -38,11 +38,15 @@ def process_output(self, output, filename, file):
         output = json.loads(output)
 
         for issue in output:
-            assert issue['startLine'] == issue['endLine']
             diff = Diff(file)
+            from_lines = issue['from'].splitlines()
+            to_lines = issue['to'].splitlines()
```

Comment:
If there is an `issue` in `output`, it will at least one line ^ ( IMHO it can't be `None`) - I checked with a file with no lint errors.